packages/vsce/src/resources/actions/TransactionActions.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { IResourceAction, IResourceContext, ResourceTypes } from "@zowe/cics-for-zowe-explorer-api";
+import { ITransaction } from "../../doc";
+
+export function getTransactionActions(): IResourceAction[] {
+  return [
+    {
+      id: "CICS.CICSLocalTransaction.INQUIRE",
+      name: "Inquire Program",
+      resourceType: ResourceTypes.CICSLocalTransaction,
+      action: "cics-extension-for-zowe.inquireProgram",
+    },
+    {
+      id: "CICS.CICSLocalTransaction.ENABLE",
+      name: "Enable Transaction",
+      resourceType: ResourceTypes.CICSLocalTransaction,
+      visibleWhen: (transaction: ITransaction, _cx: IResourceContext) => transaction.status !== "ENABLED",
+      action: "cics-extension-for-zowe.enableTransaction",
+    },
+    {
+      id: "CICS.CICSLocalTransaction.DISABLE",
+      name: "Disable Transaction",
```

Comment:
```suggestion
      name: "Disable",
```

... I'm presuming that these dictate what's shown in the menu 🤔

Reply:
Yes @davenice , keeping it same as the context menu options in tree, Although names are not getting picked from same place, but tried to keep it same as current menu options.
If you check the tree, we have `Disable Transaction` and `Enable Transaction` are the available menu option.

<img width="394" height="360" alt="Screenshot 2025-09-01 at 10 59 01 AM" src="https://github.com/user-attachments/assets/5513c5ad-181a-415d-9914-54d55d45ec42" />